sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1202,6 +1203,7 @@ nonReserved
     | DROP
     | ELSE
     | END
+    | ESCAPE
```

Comment:
we should also add it in `ansiNonReserved`

Reply:
OK.

Reply:
ah sorry I misread the document. So we expect to make `ESCAPE` to be reserved under ansi mode. This makes sense, let's change it back.

Reply:
OK. I restored it.